client/lib/user/shared-utils.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
 				'email',
 				'email_verified',
 				'is_valid_google_apps_country',
+				'user_ip_country_code',
```

Comment:
Nice find! No need to load anything this way, since `currentUser` is literally the first thing we load 👍